estela_cli/init.py

```diff
@@ -85,22 +86,34 @@ def gen_dockerfile(requirements_path):
 
 @click.command(short_help=SHORT_HELP)
 @click.argument("pid", required=True)
+@click.option(
+    "-p",
+    "--platform",
+    type=click.Choice(ALLOWED_PLATFORMS, case_sensitive=False),
+    default="scrapy",
+    help="Platform to use, it can be 'scrapy' or 'requests'",
+    show_default=True,
+)
 @click.option(
     "-r",
     "--requirements",
     default=DOCKER_DEFAULT_REQUIREMENTS,
     help="Relative path to requirements inside your project",
     show_default=True,
 )
-def estela_command(pid, requirements):
+def estela_command(pid, platform, requirements):
     """Initialize estela project
 
     PID is the project's pid
     """
-
+    platform_map = {
+        "scrapy": DOCKER_DEFAULT_ENTRYPOINT,
+        "requests": DOCKER_REQUESTS_ENTRYPOINT,
+    }
     if not os.path.exists(ESTELA_DIR):
         os.makedirs(ESTELA_DIR)
-
     estela_client = login()
-    gen_estela_yaml(estela_client, pid)
-    gen_dockerfile(requirements)
+    estela_client.update_project(pid, framework=platform.upper(), action="update")
```

Comment:
Could be useful verify this response for check everything it's OK

Reply:
Done!